www/apps/resources/app/commerce-modules/auth/persisting-auth-user/page.mdx

```diff
@@ -8,24 +8,24 @@ export const metadata = {
 
 # {metadata.title}
 
-In this document, you’ll learn what the `AuthUser` is and how to persist its authentication.
+In this document, you’ll learn what the `AuthIdentity` is and how to persist its authentication.
 
-## What is an AuthUser?
+## What is an AuthIdentity?
 
-As explained in the [Auth Provider](../auth-providers/page.mdx) guide, when a user or customer is authenticated, you receive an `authUser` object:
+As explained in the [Auth Provider](../auth-providers/page.mdx) guide, when a user or customer is authenticated, you receive an `authIdentity` object:
 
 ```ts
-const { success, authUser } =
+const { success, authIdentity } =
   await authModuleService.authenticate("emailpass", {
     // ...
   })
 ```
 
-The `authUser` object is a record of the `AuthUser` data model. It has details about the authenticated user or customer, such as their ID, email, and other details.
+The `authIdentity` object is a record of the `AuthIdentity` data model. It has details about the authenticated user or customer, such as their ID, email, and other details.
```

Comment:
We don't need to do it now, but I feel we can provide better explanation here.

The auth Identity is basically an identifier of an actor - where an actor can be user, customer, driver, a backend service, etc. It doesn't really carry any details of the actor itself (eg. first name, last name), but it is only used to identify them.

Reply:
I'm actually removing this page in another PR as it felt the details in it weren't necessary (for their original purpose). However, it would make sense to have a new document around adding new actors, etc... (once we fully support that through custom links), so would hold off on this for now

Reply:
@shahednasser you can create new actors, the old way, see: https://medusacorp.slack.com/archives/C025KMS13SA/p1716371728051229.

Now, the approach could change in the upcoming months, so I leave it up to you if it's time to document it or not 😄

Reply:
I could be misunderstanding, but isn't the way of linking the actor types (for example, driver) still in progress or done in a not-so-ideal way? If that's the case then yea I'd rather wait on documenting it. But if it's something different please let me know.

Reply:
@shahednasser it is done (eg. Victor has used it successfully in Medusa Eats), and whether it is ideal or not is something we haven't decided yet 😄

Reply:
@shahednasser yeah that's right, but for auth we are not using links at all now. Whether we'll want to switch back to links is something we'll need to decide later on, but for now you simply set eg. `user_id` in `app_metadata` on the auth identity and that's it (that acts like a link basically)

Reply:
Ahh ok I wasn't aware of that. Can you share a document or link to a slack chat around the approach now? I believe I missed it

Reply:
@shahednasser I was gonna write an internal doc on how auth works, I'll have it with you in ~30 min

Reply:
@sradevski cool no rush, I won't include it in this PR anyway 👍🏻